utils/gen-snapshot-tests.js

```diff
@@ -139,6 +139,10 @@ function extractExamplesAndComponentsForFile(sourceFile, componentFileMap) {
         src.match(/export default class \s*(\w+)/) ||
         src.match(/export default \s*\w+\(\s*(\w+)/);
 
+    if (!match) {
```

Comment:
What is this guarding against? Perhaps a comment?

Reply:
There was a file that we were traversing that didn't have a default export.